localConfig.json

```diff
@@ -326,7 +376,18 @@
               "name": "CRSSelector",
               "cfg": {
                 "additionalCRS": {
-
+                  "EPSG:2154": {
+                    "label": "EPSG:2154"
+                  },
+                  "EPSG:3945": {
+                    "label": "EPSG:3945"
+                  },
+                  "EPSG:3946": {
+                    "label": "EPSG:3946"
+                  },
+                  "EPSG:3948": {
+                    "label": "EPSG:3948"
+                  }
                 },
                 "filterAllowedCRS": [
```

Comment:
Don't you need to add the new CRS codes here also?

Reply:
@mbarto What do you mean?

Reply:
Are the new CRS visible in the combo if they are not also listed in filterAllowedCRS?

Reply:
additionalCRS is separate from filterAllowedCRS, it has not influence on it. For reference see [CoordinatesUtils](https://github.com/geosolutions-it/MapStore2/blob/43a979967a95895801d8f6cd9d30393532c082a8/web/client/utils/CoordinatesUtils.js#L429).